osu.Game.Rulesets.Mania/Mods/ManiaModConstant.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets.Mods;
+
+namespace osu.Game.Rulesets.Mania.Mods
+{
+    public class ManiaModConstant : Mod, IApplicableToBeatmap
+    {
+        public override string Name => "Constant";
+        public override string Acronym => "CT";
```

Comment:
Change name to "No SV" and change the acronym to "NSV". NSV is better recognized by mania players as an indication in versions of maps without SV.

Reply:
Don't think we do three letter acronyms anywhere. Probably won't fit in button?

Reply:
https://github.com/ppy/osu/blob/575a00dd7deafdc6a9194dec59d79737db44e0ad/osu.Game.Rulesets.Mania/Mods/ManiaModKey10.cs#L10

Reply:
Fair enough on that point - still not sure about using SV as an acronym directly in the name. Sort of inaccessible for newcomers (mania doesn't even really have "sliders").

Reply:
I like to thing mania "SV" stands for scroll velocity, but any mania player knows what SV means because it's hard not to come across the term somewhere.

The wiki details it [here](https://osu.ppy.sh/wiki/en/Guides/osu%21mania_Mapping_Guide#slider-velocity-(sv)). [A pinned thread in the osu!mania subforum](https://osu.ppy.sh/community/forums/topics/155780) uses the term for describing how to map SV. [Countless of mappers use "NSV"](https://osu.ppy.sh/beatmapsets?m=3&q=nsv&s=graveyard) in the difficulty name to indicate it's the same map without SV.

Reply:
I still think that "SV" as two letters should not be in the name, regardless of if it's established jargon. I'd be fine with "Constant Scroll" or something, as long as there are actual words there and not two-letter acronyms.

Reply:
I'm fine with "Constant Scroll" too, since gives a better idea as to what this mod does compared to just "Constant". "Constant Scrollspeed" might also work, but it might also be a bit long for a name.
Either way, I think the acronym "CT" should stay the same. "CS" is already an acronym for circle size, but that's not necessarily a mania thing, so I don't know.

Reply:
how about FS (fixed scroll)? if we're looking to avoid acronym clashes and using confusing terms like SV.